plugins/auth-backend-module-microsoft-provider/src/strategy.ts

```diff
@@ -20,11 +20,17 @@ import fetch from 'node-fetch';
 import { Strategy as MicrosoftStrategy } from 'passport-microsoft';
 
 export class ExtendedMicrosoftStrategy extends MicrosoftStrategy {
+  private shouldSkipUserProfile: boolean = false;
```

Comment:
Unfortunately I cannot name this variable `skipUserProfile` because of a method of the same name. This name was the best I could come up with.